<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateClient</class>
 <widget class="QDialog" name="CreateClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox_clientCreate">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>110</y>
     <width>491</width>
     <height>401</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushButton_clientCreate">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>327</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Criar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clientCreateCancel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>327</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>22</y>
      <width>461</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_clientCreate">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_clientName">
       <item>
        <widget class="QLabel" name="label_clientName">
         <property name="text">
          <string>Nome (*)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_clientName">
         <property name="clearButtonEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_clientEmail">
       <item>
        <widget class="QLabel" name="label_clientEmail">
         <property name="text">
          <string>Email (*)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_clientEmail"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_clientBirthDate">
       <item>
        <widget class="QLabel" name="label_clientBirthDate">
         <property name="text">
          <string>Data de nascimento</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="dateEdit_clientBirthDate"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_clientCPF">
       <item>
        <widget class="QLabel" name="label_clientCPF">
         <property name="text">
          <string>CPF</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_clientCPF"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label_clientCreateTitle">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>481</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Criar Cliente</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
